keras_cv/layers/preprocessing/random_flip_test.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+
+import numpy as np
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing.random_flip import RandomFlip
+
+
+class RandomFlipTest(tf.test.TestCase):
+    def _run_test(self, mode, expected_output=None, mock_random=None):
+        np.random.seed(1337)
+        num_samples = 2
+        orig_height = 5
+        orig_width = 8
+        channels = 3
+        if mock_random is None:
```

Comment:
tests shouldn't rely so heavily on default values like this.  Remove the default values for both this and expected_output

Reply:
done!